### PR TITLE
eth/downloader, les/downloader: fix log errors of queue_test.go

### PR DESCRIPTION
**Pull request by int88**

### CODE REVIEW

File: eth/downloader/queue_test.go [hunk 1]
```diff
@@ -249,9 +249,9 @@ func TestEmptyBlocks(t *testing.T) {
 		peer := dummyPeer("peer-3")
 		fetchReq, _, _ := q.ReserveReceipts(peer, 50)
 
-		// there should be nothing to fetch, blocks are empty
+		// there should be nothing to fetch, receipts are empty
```
Comment (karalabe) [inline eth/downloader/queue_test.go:252]: This was correct, the block is empty, hence why there are no receipts

File: les/downloader/queue_test.go [hunk 2]
```diff
@@ -237,9 +237,9 @@ func TestEmptyBlocks(t *testing.T) {
 		peer := dummyPeer("peer-3")
 		fetchReq, _, _ := q.ReserveReceipts(peer, 50)
 
-		// there should be nothing to fetch, blocks are empty
+		// there should be nothing to fetch, receipts are empty
```
Comment (karalabe) [inline les/downloader/queue_test.go:240]: Same, it was correct

**Comment by int88** [comment]
@karalabe updated.

**Review by karalabe — APPROVED** [review]
Thanks

**Pull request merged**